src/EditorFeatures/CSharpTest2/Recommendations/ReadOnlyKeywordRecommenderTests.cs

```diff
@@ -394,10 +394,9 @@ public class ReadOnlyKeywordRecommenderTests : KeywordRecommenderTests
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotAfterNew()
+        public async Task TestAfterNew()
         {
-            await VerifyAbsenceAsync(
-@"new $$");
+            await VerifyKeywordAsync(SourceCodeKind.Regular, @"new $$");
```

Comment:
but this test is still a little odd.. this code is incorrect outside a class/struct

Reply:
@Neme12 It is not a requirement for completion to strictly only offer was is allowed. It's better to offer a bit too much than bit too little.

Reply:
@jcouv I didn't mean that you have to change the behavior and add a test to make sure it's not offered here. I just think it would be better to test this in a scenario where it makes sense (inside a class). I don't care if it's offered here or not. But I know we *do* care it's offered inside a class after new.

Reply:
Thanks for clarifying.
What's the scenario where `new readonly` is useful inside a class?

Reply:
```c#
class B
{
    protected struct S { }
}

class C : B
{
    new readonly struct S { }
}
```

Reply:
```c#
class OuterClass
{
    public struct InnerStruct
    {

    }

    class InnerClass : OuterClass
    {
        public new readonly struct InnerStruct
        {

        }
    }
}
```

Reply:
Note: this is *super* corner case.  i personally wouldn't care much if this didn't happen, or came later.

Reply:
Thanks. I expect it will just work. I'll add a test